### PR TITLE
Expose --excludes in the command line API.

### PR DESCRIPTION
**Pull request by swankjesse**

### CODE REVIEW

File: wire-compiler/src/main/java/com/squareup/wire/CommandLineOptions.java [hunk 1]
```diff
@@ -28,6 +28,8 @@
   public static final String JAVA_OUT_FLAG = "--java_out=";
   public static final String FILES_FLAG = "--files=";
   public static final String ROOTS_FLAG = "--roots=";
```
Comment (swankjesse) [inline wire-compiler/src/main/java/com/squareup/wire/CommandLineOptions.java]: roots is now a non-documented feature of the command line API

Comment (swankjesse) [inline wire-compiler/src/main/java/com/squareup/wire/CommandLineOptions.java]: I'll delete it completely, but not urgently 'cause now we have plugins that expect this to exist

File: wire-compiler/src/main/java/com/squareup/wire/CommandLineOptions.java [hunk 2]
```diff
@@ -64,7 +66,8 @@
    * <pre>
    * java WireCompiler --proto_path=&lt;path&gt; --java_out=&lt;path&gt;
```
Comment (JakeWharton) [inline wire-compiler/src/main/java/com/squareup/wire/CommandLineOptions.java]: All this Javadoc here is completely useless as this isn't a public type. We should merge this class with `WireCompiler`.

Comment (JakeWharton) [inline wire-compiler/src/main/java/com/squareup/wire/CommandLineOptions.java]: Maybe rename it `WireCommandLine` or `WireCli` while we're at it

Comment (JakeWharton) [inline wire-compiler/src/main/java/com/squareup/wire/CommandLineOptions.java]: Or just `com.squareup.wire.compiler.Main`

File: wire-compiler/src/main/java/com/squareup/wire/CommandLineOptions.java (hunk 1 above)
Comment (JakeWharton) [inline wire-compiler/src/main/java/com/squareup/wire/CommandLineOptions.java]: Mark with a TODO to remove before 2.0 final?

File: wire-compiler/src/main/java/com/squareup/wire/CommandLineOptions.java (hunk 2 above)
Comment (swankjesse) [inline wire-compiler/src/main/java/com/squareup/wire/CommandLineOptions.java]: https://github.com/square/wire/pull/484

**Pull request merged**